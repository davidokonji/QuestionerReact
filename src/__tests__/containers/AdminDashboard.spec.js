import React from 'react';
import { shallow } from 'enzyme';
import { AdminDashboard } from '../../containers/AdminDashboard.container';

const meetup = {
  meetups: {
    data: []
  }
};

const allMeetups = jest.fn();

describe('Admin container', () => {
  it('should render container correctly', () => {
    const wrapper = shallow(<AdminDashboard meetups={meetup} allMeetups={allMeetups} />);
    expect(wrapper).toMatchSnapshot();
  });
});
