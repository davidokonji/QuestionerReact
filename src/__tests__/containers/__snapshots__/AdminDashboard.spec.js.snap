// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin container should render container correctly 1`] = `
<div>
  <ConnectFunction />
  <div
    className="container"
  >
    <div
      className="row mt-5"
    >
      <ConnectFunction
        open={false}
        toggle={[Function]}
      />
      <AdminTable
        data={Array []}
      />
    </div>
  </div>
  <Footer />
</div>
`;
